test/library/folder-variants.test.js

```diff
@@ -30,6 +30,31 @@ describe('folder variants', function () {
         });
     });
 
+    it('should not preserve the order if not specified', function (done) {
+        newman.run({
+            collection: collection,
+            folder: ['R3', 'R1'],
+            preserveOrder: false
+        }, function (err, summary) {
+            expect(err).to.be.null;
+            expect(summary.run.executions.map((e) => { return e.item.name; })).to.eql(['R1', 'R3']);
+            done();
+        });
+    });
+
+    // eslint-disable-next-line mocha/no-skipped-tests
+    it.skip('should preserve the order if specified', function (done) {
```

Comment:
It is so because the correct order depends on the release of the next version of postman-runtime.
The corresponding request there has been merged.
So we can run this test when it is released.